service/awsconfig/v4/key/key.go

```diff
@@ -17,9 +17,6 @@ const (
 	// and managed by a cluster.
 	CloudProviderTagOwnedValue = "owned"
 
-	// LegacyVersion is the version in the version bundle for existing clusters.
-	LegacyVersion = "0.1.0"
-
```

Comment:
This here is dead code.

Reply:
Good point. This is only needed by the legacy resource in v1.